index.html

```diff
@@ -10,17 +10,29 @@
     <div class="mobile-container">
         <header class="top">
         <a class="link-name" href="#">Shuhad Loofer</a>
-        <nav class="nav">
-            <a><img src="./images/Menu.svg" alt="Menu"/></a>
-        </nav>
+        <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
         <nav class="nav1">
-            <a href="#">Profile</a>
-            <a href="#">About</a>
-            <a href="#">Contact</a>
-            <a href="#"><img src="./images/Union.svg" alt="Email"/></a>
+            <ul class="nav-menu" >
+                <li class="nav-item">
+                    <a class="nav-link" href="#Profile">Profile</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#About">About</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#Contact">Contact</a>
+                </li>
+                <li class="nav-email" class="nav-item">
+                    <a href="#"><img src="./images/Union.svg" alt="Email"/></a>
```

Comment:
- [ ] [OPTIONAL] It will be better if you add mobile menu icon inside nav tag instead of out it to make it better like the design and when scroll not appear